run.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+if [ "${POSTGRES_ENV_POSTGRES_PASSWORD}" == "**Random**" ]; then
```

Comment:
what's this for?

Reply:
Bash script is based on first approach with backup, I'll remove it 👍